src/main/resources/messages_es.properties

```diff
@@ -960,11 +960,12 @@ ethnicity-selection.option2=No hispano ni latino
 race-selection.title=Selecci�n de raza
 race-selection.header=�Con qu� razas se identifica?
 race-selection.subheader=Seleccione todas las opciones que correspondan.
-race-selection.option1=Ind�gena americana o nativa de Alaska
-race-selection.option2=Asi�tica
-race-selection.option3=Negra o afroamericana
-race-selection.option4=Nativa de Haw�i o de otras islas del Pac�fico
-race-selection.option5=Cauc�sica
+race-selection.option1=Ind�gena americana
+race-selection.option2=Nativa de Alaska
```

Comment:
This and the VI translations need to be double checked or overwritten by transifex. Not sure how that process works.

Reply:
Yeah, once this is merged into main, we can double check it in [transifex](https://app.transifex.com/code-for-america-1/snil-la-doc-upload/translate/#es/src-main-resources-messages-properties--main/475498028?q=key%3Arace)